### PR TITLE
Add an option var to change the hawkular hostname

### PR DESCRIPTION
**Pull request by yaacov**
Add an optional variable to set the hawkular metrics hostname, defaults to current value.

the haawkular metrics hostname defaults to the current value: `hawkular-metrics.{{ openshift.master.default_subdomain }}`.

Issue:
https://github.com/openshift/openshift-ansible/issues/2208

### CODE REVIEW

**Comment by yaacov** [comment]
@alongoldboim @simon3z please review

**Comment by alongoldboim** [comment]
LGTM :+1:

**Comment by simon3z** [comment]
I have the feeling that the previous `hawkular-metrics.{{ openshift.master.default_subdomain }}` should be maintained as default (when not otherwise specified).

**Comment by yaacov** [comment]
@simon3z :+1:  the hawkular metrics hostname now defaults to the old value, if the user does not specify a hostname.

**Comment by simon3z** [comment]
@yaacov :+1:

**Comment by yaacov** [comment]
@enoodle please review

File: roles/openshift_metrics/README.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ From this role:
 | Name                                            | Default value         |                                                             |
 |-------------------------------------------------|-----------------------|-------------------------------------------------------------|
 | openshift_hosted_metrics_deploy                 | `False`               | If metrics should be deployed                               |
+| hawkular_metrics_hostname                       | `hawkular-metrics.<default_subdomain>` | Metrics hostanme                       |
```
Comment (simon3z) [inline roles/openshift_metrics/README.md]: @yaacov this seems to break the table, isn't it?

Comment (yaacov) [inline roles/openshift_metrics/README.md]: a. The mark down is parsed correctly, I did not want to edit the other lines and move the `|` mark ...
b. Added a new commit to fix the align :-)

**Comment by enoodle** [comment]
LGTM :+1:

**Comment by sdodson** [comment]
The metrics portion is currently being largely re-written and includes similar changes. I hope to have that pushed up for review today, lets wait to see if that meets your needs.

**Comment by yaacov** [comment]
> The metrics portion is currently being largely re-written

Closing this PR as the metrics role is under development.

@sdodson @alongoldboim will be happy to re-test with manageIQ metrics when you feel it is ready for testing.

**Pull request closed without merging**